_jekyll/layouts/video.html

```diff
@@ -67,14 +67,27 @@
     {% comment %} List of community contributions (only visible if can_contribute is true). {% endcomment %}
     {% if page.can_contribute %}
       <div class="contributions">
+        {% comment %} Converts page.video_number to a string. {% endcomment %}
+        {% capture video_number %}{{ page.video_number }}{% endcapture %}
+
+        {% comment %} Set video which contains the contributions. {% endcomment %}
+        {% comment %} Multi-part videos allow contributions only on the first part. {% endcomment %} 
+        {% if video_number contains '.' and page.contributions == null %}
```

Comment:
What's the purpose of the `page.contributions == null` check here? If we want to enforce this can we not just ignore the current contributions (And I can lint them as invalid and move invalid ones to their master file)

Reply:
I thought that it would be nice if we can still have contributions on 'sub-parts' when needed. For example, a challenge might have five parts where part one to four is the challenge itself and part five is a variation of the original challenge made by @shiffman.

Reply:
Is there an example of this being needed for a particular multi-part challenge at the moment? Might be easier to maintain if we keep it simple and add this feature if it turns out we need it later? I'm ok either way and defer to @nilsweb and @meiamsome.

Reply:
I think it makes sense that the Jekyll side be programmed for this case either way. We can enforce this in tests instead.